<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="org.springframework.web.context.request.async.WebAsyncManagerTimeoutTests" tests="9" skipped="0" failures="0" errors="0" timestamp="2016-11-10T08:05:26" hostname="XB-20161010YVDY" time="0.011">
  <properties/>
  <testcase name="startCallableProcessingTimeoutAndResumeThroughInterceptor" classname="org.springframework.web.context.request.async.WebAsyncManagerTimeoutTests" time="0.001"/>
  <testcase name="startCallableProcessingAfterTimeoutException" classname="org.springframework.web.context.request.async.WebAsyncManagerTimeoutTests" time="0.002"/>
  <testcase name="startDeferredResultProcessingTimeoutAndResumeThroughCallback" classname="org.springframework.web.context.request.async.WebAsyncManagerTimeoutTests" time="0.003"/>
  <testcase name="startCallableProcessingTimeoutAndResumeThroughCallback" classname="org.springframework.web.context.request.async.WebAsyncManagerTimeoutTests" time="0.001"/>
  <testcase name="startCallableProcessingTimeoutAndComplete" classname="org.springframework.web.context.request.async.WebAsyncManagerTimeoutTests" time="0.001"/>
  <testcase name="startDeferredResultProcessingAfterTimeoutException" classname="org.springframework.web.context.request.async.WebAsyncManagerTimeoutTests" time="0.002"/>
  <testcase name="startDeferredResultProcessingTimeoutAndResumeThroughInterceptor" classname="org.springframework.web.context.request.async.WebAsyncManagerTimeoutTests" time="0.0"/>
  <testcase name="startDeferredResultProcessingTimeoutAndComplete" classname="org.springframework.web.context.request.async.WebAsyncManagerTimeoutTests" time="0.001"/>
  <testcase name="startDeferredResultProcessingTimeoutAndResumeWithDefaultResult" classname="org.springframework.web.context.request.async.WebAsyncManagerTimeoutTests" time="0.0"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
